docs/source/models/intervention_models/alzheimers/hypothetical_treatment.rst

```diff
@@ -30,4 +30,232 @@
 
 ========================================
 Hypothetical Alzheimer's Treatment
-========================================
\ No newline at end of file
+========================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - BBBM
+    - Blood-based biomarker
+    - BBBM tests measure blood plasma protein levels and are less invasive than CSF (Cerebrospinal Fluid) or PET (Positron Emission Topography) tests
+  * - MCI
+    - Mild cognitive impairment
+    -
+
+Intervention Overview
+-----------------------
+
+The hypothetical treatment intervention is triggered by a positive BBBM test, and has the effect slowing the progression
+from pre-clinical to MCI state. This progression rate is set by a :ref:`time-dependent hazard function <2021_cause_alzheimers_presymptomatic_mci_transition_data_table>`
+which is multiplied by a hazard ratio H < 1 when a simulant has an active treatment effect in order to slow the progression.
+This effect can wane over time (udpated each time step) and when the effect fully expires, H returns to 1. 
+
+This treatment is hypothetical and we don't have confirmed information about the mechanism.
+
+
+.. list-table:: Affected Outcomes
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note (ex: is this relationship direct or mediated?)
+  * - Hypothetical treatment effect
+    - Slows progression to MCI (time-dependent effect size)
+    - Yes
+    - 
+
+
+
+Vivarium Modeling Strategy
+--------------------------
+
+.. graphviz::
+
+    digraph treatment_states {
+        rankdir = TB;
+        el [label="BBBM test eligible [variable]"]
+        test [label="BBBM test received", style=dashed, shape=box]
+
+        pos [label="BBBM test positive", style=dashed, shape=box]
+        neg [label="BBBM test negative [3 y]"]
+
+        wait [label="In treatment/ Waiting for treatment [6 mo]"]
+        no_treat [label="No treatment effect [permanent]"]
+
+        treat [label="Full treatment effect LONG [5 y]"]
+        wane [label="Waning treatment effect LONG (update each step) [9 y]"]
+
+        treat_short [label = "Full treatment effect SHORT [6 mo]"]
+        wane_short [label = "Waning treatment effect SHORT (update each step) [2.5 y]"]
+
+
+        el -> test [label = "tested"]
+        el -> el [label = "not tested"]
+
+        test -> pos  [label = "(90%)", style=dashed]
```

Comment:
So everyone who starts treatment either does it for 6 months OR the full 5 years? Nothing in between?